crates/sui-graphql-rpc/src/types/move_package.rs

```diff
@@ -54,6 +54,8 @@ pub(crate) struct MovePackageDowncastError;
 
 pub(crate) type CModule = Cursor<String>;
 
+/// The representation of a Move Package, which provides more information about previous package
+/// versions, type layouts, the modules defined in this package and others.
```

Comment:
`MovePackage` doesn't (yet) return information about previous package versions. 

```suggestion
/// A MovePackage is a kind of Move object that represents code that has been published on chain. It exposes
/// information about its modules, type definitions, functions, and dependencies.
```